public/app/percona/shared/services/updates/Updates.service.ts

```diff
@@ -1,8 +1,21 @@
+import {
+  CheckUpdatesChangeLogsResponse,
+  SnoozePayloadBody,
+  SnoozePayloadResponse,
+} from 'app/percona/shared/core/reducers/updates';
+
 import { api } from '../../helpers/api';
 
 import { CheckUpdatesBody, CheckUpdatesResponse } from './Updates.types';
 
 export const UpdatesService = {
   getCurrentVersion: (body: CheckUpdatesBody = { force: false }) =>
-    api.post<CheckUpdatesResponse, CheckUpdatesBody>('/v1/Updates/Check', body, true),
+    api.get<CheckUpdatesResponse, CheckUpdatesBody>('/v1/server/updates', false, { params: body }),
+
+  getUpdatesChangelogs: () => api.get<CheckUpdatesChangeLogsResponse, void>('/v1/server/updates/changelogs', false),
+
+  setSnoozeCurrentVersion: (body: SnoozePayloadBody) =>
```

Comment:
APIs for `snooze` are already part of the `User.service`, no need to add them here.

I think the PUT `/v1/users/me` can be updated partially (just send field for the snooze version)

Reply:
fixed